template/node12/index.js

```diff
@@ -61,23 +63,29 @@ function handleAppResource(req, res) {
     }
 }
 
+async function initManifest() {
+    if (manifest == null) {
+        let possibleFutureRes = manifestHandler();
+        let tempManifest = await possibleFutureRes;
```

Comment:
Can be simplified by : 
```suggestion
        let tempManifest = await manifestHandler();
```